lib/burnex.ex

```diff
@@ -71,51 +85,40 @@ defmodule Burnex do
     end
   end
 
+  def is_burner_domain?(_), do: true
+
```

Comment:
This could be considered a breaking change as invalid parameters will now return `true` rather than crashing. We can discuss this idea in a separate issue, but I prefer to remote it from the PR for now.